<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="backgroundVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Connections</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="disconnectkuka">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect robot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectSTM32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect STM32F4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectAurora">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect Aurora NI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectKuka">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Kuka robot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectSTM32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect STM32F4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectAurora">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Aurora NI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boot_Aurora">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Aurora Boot</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Measurements</string>
    </property>
    <widget class="QPushButton" name="startMeasure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Measurements</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopMeasure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Measurements</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>460</y>
      <width>560</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Robot Coordenates</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>36</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>YAW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>36</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PITCH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>36</y>
       <width>36</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AXIS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>104</x>
       <y>30</y>
       <width>17</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>202</x>
       <y>30</y>
       <width>16</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>16</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>95</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Maximum</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>126</y>
       <width>39</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STEP</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="z_min">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>63</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="y_min">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>63</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_min">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>63</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="pitch_min">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>63</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-400</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="yaw_min">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>63</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-400</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_max">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>93</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="y_max">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>93</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="z_max">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>93</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-99</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="yaw_max">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>93</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-400</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="pitch_max">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>93</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-400</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_inc">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>123</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="y_inc">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>123</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="z_inc">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>123</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="yaw_step">
     <property name="geometry">
      <rect>
       <x>382</x>
       <y>123</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="pitch_inc">
     <property name="geometry">
      <rect>
       <x>483</x>
       <y>123</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>66</y>
       <width>68</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Minimum</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeApp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>14</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Events</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="upARMrobot">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>24</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Up position Robot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="requestdataSTM32">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>54</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Request data STM32</string>
     </property>
    </widget>
    <widget class="QPushButton" name="requestDataAurora">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Request data Aurora</string>
     </property>
    </widget>
    <widget class="QPushButton" name="STOP_tracking">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>140</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP Tracking Mode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="START_tracking">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>80</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>START Tracking Mode</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
